data/recommendation/notify.js

```diff
@@ -1,14 +1,21 @@
 class Notify {
   start() {
     this.setup();
-    self.port.on('data', (/* recs */) => { // TODO: use rec data for customized panel info
-      this.showAdvice();
+    self.port.on('data', recs => {
+      this.showAdvice(recs);
     });
   }
 
-  showAdvice() {
+  showAdvice(recs) {
+    const date = new Date(recs.BreachDate);
+    const year = date.getFullYear();
+    const victimCount = recs.PwnCount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
It is probably worth moving this to a helper function named something like `prettyPrintNumber` or something. That's a pretty intense regex to not have a name attached to it.